script/script.go

```diff
@@ -385,12 +422,16 @@ func setRequestValue(f filters.FilterContext) func(*lua.LState) int {
 		case "url":
 			u, err := url.Parse(s.ToString(-1))
 			if err != nil {
-				s.Push(lua.LString(err.Error()))
-				return 1
+				s.RaiseError("%v", err)
```

Comment:
@szuecs Here is another breaking change. Before the change it was a no-op in case of failure and also returned string error which is not possible to use in lua since assignment operator does not return value.

Reply:
For now I have just added a test until we decide how to deal with backward compatibility which I guess depends on the size of lua userbase

Reply:
Let's add a TODO as commented in https://github.com/zalando/skipper/pull/1485#discussion_r453818921